src/components/Navbar/styles.module.scss

```diff
@@ -1,4 +1,5 @@
 @import '../../scss/variables';
+@import '../../scss/mixins';
```

Comment:
I don't see any new mixins used. Is this still needed?

Reply:
Ah, no, I removed the mixin that I added.